tests/integration/views/test_static.py

```diff
@@ -0,0 +1,24 @@
+import mock
+from flask import json
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+with open('data/en/1_0205.json') as json_data:
+    data = json.load(json_data)
+
+
+class TestStatic(IntegrationTestCase):
+    @mock.patch('app.utilities.schema.load_schema_from_params')
+    def test_contact(self, mock_contact):
+        mock_contact.return_value = data
+        self.launchSurvey('test', '0205')
+        self.get('/contact-us')
+        self.assertInPage('123456789012A')
+
+    def test_contact_with_no_metadata(self):
+        self.get('/contact-us')
+        self.assertInPage('Sorry, something has gone wrong.')
```

Comment:
This shouldn't error. it should show the page without the details from the session